{"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/IOOnClosedConnection\/location"},"kind":"symbol","primaryContentSections":[{"declarations":[{"languages":["swift"],"tokens":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"location","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"text":" { ","kind":"text"},{"text":"get","kind":"keyword"},{"text":" }","kind":"text"}],"platforms":["Linux"]}],"kind":"declarations"}],"abstract":[{"type":"text","text":"The location where the error was thrown."}],"hierarchy":{"paths":[["doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2","doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors","doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/IOOnClosedConnection"]]},"sections":[],"variants":[{"paths":["\/documentation\/niohttp2\/niohttp2errors\/ioonclosedconnection\/location"],"traits":[{"interfaceLanguage":"swift"}]}],"metadata":{"role":"symbol","roleHeading":"Instance Property","modules":[{"name":"NIOHTTP2"}],"externalID":"s:8NIOHTTP20A6ErrorsO20IOOnClosedConnectionV8locationSSvp","title":"location","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"location","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"}],"symbolKind":"property"},"schemaVersion":{"patch":0,"major":0,"minor":3},"references":{"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/NIOHTTP2Errors/IOOnClosedConnection":{"type":"topic","url":"\/documentation\/niohttp2\/niohttp2errors\/ioonclosedconnection","abstract":[{"text":"An attempt to perform I\/O was made on a connection that is already closed.","type":"text"}],"fragments":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"text":"IOOnClosedConnection","kind":"identifier"}],"navigatorTitle":[{"text":"IOOnClosedConnection","kind":"identifier"}],"role":"symbol","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/IOOnClosedConnection","kind":"symbol","title":"NIOHTTP2Errors.IOOnClosedConnection"},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/NIOHTTP2Errors/IOOnClosedConnection/location":{"type":"topic","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/IOOnClosedConnection\/location","kind":"symbol","abstract":[{"type":"text","text":"The location where the error was thrown."}],"role":"symbol","url":"\/documentation\/niohttp2\/niohttp2errors\/ioonclosedconnection\/location","title":"location","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"location"},{"kind":"text","text":": "},{"preciseIdentifier":"s:SS","kind":"typeIdentifier","text":"String"}]},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2":{"type":"topic","url":"\/documentation\/niohttp2","kind":"symbol","title":"NIOHTTP2","abstract":[],"role":"collection","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2"},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/NIOHTTP2Errors":{"abstract":[{"type":"text","text":"Errors that NIO raises when handling HTTP\/2 connections."}],"fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"NIOHTTP2Errors"}],"role":"symbol","type":"topic","url":"\/documentation\/niohttp2\/niohttp2errors","kind":"symbol","navigatorTitle":[{"kind":"identifier","text":"NIOHTTP2Errors"}],"identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors","title":"NIOHTTP2Errors"}}}